tests/phpunit/tests/Options/Options/test-OffsetUnset.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace WP_Syntex\Polylang\Tests\Integration\Options\Options;
+
+use PLL_UnitTestCase;
+
+/**
+ * Tests for `Options\Options->offsetUnset()`.
+ *
+ * @group options
+ */
+class OffsetUnset_Test extends PLL_UnitTestCase {
+	/**
+	 * Makes sure that `offsetUnset()` doesn't modify the option's value (because `offsetGet()` doesn't return a reference).
```

Comment:
Could we make `offsetUnset()` behave as usual even though `offsetGet()` doesn't return a reference?

Reply:
No, the issue is the same: child's `offsetUnset()` won't trigger parent's `offsetUnset()`.